pkgs/shells/zsh/zinit/default.nix

```
@@ -24,6 +24,9 @@ stdenvNoCC.mkDerivation rec {
     install -m0644 zinit{,-side,-install,-autoload}.zsh "$outdir"
     install -m0755 share/git-process-output.zsh "$outdir"
 
+    mkdir "$outdir/doc"
+    install -m0644 doc/zinit.1 "$outdir/doc/zinit.1"
```

Use `installManpage` in `installShellFiles` (add it to `nativeBuildInputs`).

```suggestion
    installManpage doc/zinit.1
```

---

I had to update it to `installManPage` [docs](https://nixos.org/manual/nixpkgs/stable/#installshellfiles) but this is done :)